README.md

```diff
@@ -1,16 +1,33 @@
 # Connectors
-Connectors allows investors to provide liquidity into Centrifuge pools without having to bridge over to the Centrifuge chain. It provides a fully native experience for users on any supported chain.
+Connectors enables seamless deployment of Centrifuge RWA pools on any EVM-compatible blockchain.
 
+## How it works
+<a href="https://github.com/centrifuge/connectors">
+  <img alt="Centrifuge" src="contracts.png">
```

Comment:
This new image is ~159kb. If we run it through https://tinypng.com/ it can become 40.7kb. Even better would be to use it as a svg file - can we do that?

Reply:
Just tried SVG but this happens:

<img width="797" alt="Screenshot 2023-05-01 at 09 49 13" src="https://user-images.githubusercontent.com/1748621/235425009-46ba7d54-ed1e-4966-836a-8828178e9a83.png">

Will change it to a compressed png

Reply:
I guess we would need to export the svg with the option of exporting text as a path and not as a font (my graphics editor of choice has that option). But this is fine 👍🏻

Reply:
I guess we would need to export the svg with the option of exporting text as a path and not as a font (my graphics editor of choice has that option). But this is fine 👍🏻

Reply:
Good idea, but unfortunately not supported by Excalidraw it seems